src/commands/clear/index.ts

```diff
@@ -0,0 +1,105 @@
+import { Spinner } from "../../lib/index.js";
+import { SwankyCommand } from "../../lib/swankyCommand.js";
+import { ConfigError, FileError } from "../../lib/errors.js";
+import fs from "fs-extra"
+import path from "path"
+import { Args, Flags } from "@oclif/core";
+import { flagUsage } from "@oclif/core/lib/parser/help.js";
+
+type Folder = {name: string, contractName?: string, path: string}
+
+export default class Clear extends SwankyCommand<typeof Clear> {
+
+  static flags = {
+    all: Flags.boolean({
+      char: "a",
+      description: "Select all the project artifacts for delete"
+    }),
+  };
+
+  static args = {
+    contractName: Args.string({
+      name: "contractName",
+      required: false,
+      description: "Name of the contract artifact to clear",
+    }),
+  };
+
+  static async deleteFolder(path: string): Promise<void> {
+    try {
+      await fs.remove(path);
+    } catch (err) {
+      throw new FileError(
+        `Error while to delete the folder ${path}: ${err}Error while to delete the folder ${path}: ${err}`
```

Comment:
Why the repetition?
When rethrowing, the orginal error can be included as a second arguement's `cause` property.
`throw new FileError("Error deleting the folder ${path}.", {cause: err})`